src/Model/Command/UserRecommendation.php

```diff
@@ -173,6 +175,20 @@ public function setModelName(string $modelName): self
         return $this;
     }
 
+    /**
+     * Allow items, that the user has already "seen"
+     *
+     * By default user won't see any items, that he or she has visitted (and we have recorded DetailView interaction.)
+     * If you want to circumvent this, and get recommendations including the ones, that the user has already visitted,
+     * you can set the "seen" allowance here.
+     */
+    public function setAllowSeen(bool $seen = true): self
```

Comment:
This is confusing - we should either has setter `setAllowSeen(bool)` with **required** parameter, or two methods to set it to true and false (`allowSeen()` and eg. `disallowSeen()`). For simplicity I'd go for the first variant, but for sure not mix those two approaches.